### ISSUE TITLE
Allow blocking of users from suggesting

### ISSUE THREAD

**Issue by phlax** [translate/pootle#5290]
If a user persistently creates bad suggestions (eg they just use machine translation) there should be a way for language teams to block the user

**Comment by unho** [comment]
Perhaps this can be taken of by using the negative permissions. Needs a bit of thought on how to expose this to admins, since probably they would want to just ban the user without specifying specific permissions. If this is added we probably need a better UI to list "who can and who cannot do what".

**Comment by phlax** [comment]
yep, we might also want to give the admin a way of notifying the user
